package.json

```diff
@@ -18,7 +18,7 @@
     "@mate-academy/cypress-tools": "^1.0.4",
     "@mate-academy/eslint-config-react": "0.0.7",
     "@mate-academy/eslint-config-react-typescript": "^1.0.1",
-    "@mate-academy/scripts": "^1.2.1",
+    "@mate-academy/scripts": "^1.2.3",
```

Comment:
You don't need to change those versions each time. ^1.2.1 will accept all 1.2.* including 1.2.3. 
It is correct ^ is fine in most cases, however better and saver solution is to use ~ instead.

^1.2.1 will accept 1.3.1 version, there can be breaking changes. 
~1.2.1 will accept only 1.2.* versions, so you can avoid breaking changes. 

[npm versioning](https://www.geeksforgeeks.org/difference-between-tilde-and-caret-in-package-json/)
[see this](https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json)